networkit/cpp/graph/GraphTools.cpp

```diff
@@ -200,6 +200,19 @@ double density(const Graph &G) noexcept {
     return m / (n * (n - 1));
 }
 
+double volume(const Graph &G) {
+    if (G.isDirected())
+        WARN("Commute time distance is only supported for undirected graphs.");
+
+    double volG = 0.0;
+    if (!G.isWeighted()) {
+        volG = 2.0 * G.numberOfEdges();
+    } else {
+        volG = 2.0 * G.totalEdgeWeight();
+    }
+    return volG;
```

Comment:
`volG` is not needed, we can simply replace the assignments with `return`.

Reply:
Wouldn't it be nicer to have a more general method for a Graph G and a subset of the nodes V'? Volumes of subsets are of frequent interest, e.g. in clustering algorithms.

Reply:
Sounds like a good idea. Since we want the fix in the release and this also includes new unit-tests and notebook examples, the subset-volume will be handled in a different PR.